user/configure-device.rst

```diff
@@ -106,7 +111,9 @@ You will see the default config file, something like the following:
 
 Uncomment and change the following fields:
 
-- ``url``: the hostname of your OpenWISP2 controller
+- ``url``: the hostname of your OpenWISP2 controller (For example, if you are hosting your OpenWISP
+  server locally and you set the IP Address to "192.168.56.2", the url would be "https://192.168.56.2/".
+  Note that the "/admin" after the IP Address does not need to be added in the url.)
```

Comment:
this line can be misleading and it's unnecessary, I think you should remove it.

Reply:
I didn't remove the line but rather removed the last part of it to make it less confusing.